codemeta.json

```diff
@@ -289,6 +289,23 @@
             "affiliation": "Hylleraas Centre for Quantum Molecular Sciences, UiT -- The Arctic University of Norway",
             "@id": "https://orcid.org/0000-0002-5452-9239",
             "identifier": "https://github.com/robertodr"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Maximilian",
+            "familyName": "Scheurer",
+            "affiliation": "Interdisciplinary Center for Scientific Computing, Heidelberg University",
+            "@id": "https://orcid.org/0000-0003-0592-3464",
+            "identifier": "https://github.com/maxscheurer"
+        },
+        {
+            "@type": "Person",
+            "givenName": "Michael",
+            "additionalName": "F.",
+            "familyName": "Herbst",
+            "affiliation": "Inria Paris and CERMICS, École des Ponts ParisTech",
```

Comment:
```suggestion
            "affiliation": "Inria Paris and CERMICS, \u0137cole des Ponts ParisTech",
```

Reply:
Done.

Reply:
I think "\u00C9" is correct, though 😄

Reply:
Yes please, no funny ks.

Reply:
looks like they're the same https://en.wikipedia.org/wiki/List_of_Unicode_characters. I chose from the column the psi4 codemeta used. I don't actually recall what software balked at direct characters anyways at this point. :-)

Reply:
```
>>> x = u"\u0137"
>>> x
'ķ'
```
on my machine 😄

Reply:
Same here :smile: